charmcraft/commands/build.py

```diff
@@ -109,11 +111,20 @@ def _link_to_buildpath(self, srcpath):
         destpath.symlink_to(srcpath)
         return destpath
 
+    def _load_juju_ignore(self):
+        ignore = JujuIgnore(default_juju_ignore)
+        path = self.charmdir / '.jujuignore'
+        if path.exists():
+            with path.open() as ignores:
```

Comment:
Shouldn't we specify the encoding here? Which is the expected encoding for the `.jujuignore` file?

Reply:
.gitignore is officially UTF-8, I don't know that Juju specifically declares an encoding, but there isn't any reason to believe it anything but UTF-8.